packages/cli/src/modules/breaking-changes/types/rule.types.ts

```diff
@@ -0,0 +1,78 @@
+import type { WorkflowEntity } from '@n8n/db';
+
+export const enum BreakingChangeSeverity {
+	CRITICAL = 'critical',
```

Comment:
The typical convention for enum keys in TS is to use PascalCase instead of ALL CAPS